### ISSUE TITLE
Allow tabs for indentation

### ISSUE THREAD

**Issue by seitokaichou** [michaelforney/samurai#26]
Currently, attempting to indent `key = value` pairs with tabs causes `expected name` errors, which is counter-intuitive. Especially considering that tabs are valid whitespace for line continuation.

The Ninja devs seem to be ignoring this (https://github.com/ninja-build/ninja/issues/1598).

If there is a good reason to reject this, please consider raising an appropriate error message.

Thank you.

**Comment by michaelforney** [comment]
Yes, I think that would be quite nice to have, and I see no reason why allowing them would be a problem. Although scripts that use tabs would be incompatible with ninja.

This should be an easy change.

**Comment by pkitszel** [comment]
Let's see how ninja will receive my PR
https://github.com/ninja-build/ninja/pull/2392
it's just lexer change at the moment
